src/homepageExperience/components/steps/arduino/ExecuteQuery.tsx

```diff
@@ -30,5 +84,19 @@ export const ExecuteQuery = (props: OwnProps) => {
     return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
   }, [])
 
-  return <h1>Execute a Flux Query on {bucket}</h1>
+  return (
+    <>
+      <h1>Execute a Flux Query on {bucket}</h1>
+      <p className="small-margins">
+        In this query, we are looking for data points within the last 1 minute
+        with a measurement of{' '}
+        <code className="homepage-wizard--code-highlight">"wifi_status"</code>.
+      </p>
+      <CodeSnippet
+        text={codeSnippet}
+        onCopy={logCopyCodeSnippet}
+        language="properties"
```

Comment:
Also, since this is C language, can you try changing this to C and see how the colors do? 

you can check out all available themes here:


https://react-syntax-highlighter.github.io/react-syntax-highlighter/demo/